challenges/2017-08-10-radio-transmitters/README.md

```diff
@@ -0,0 +1,7 @@
+# Radio Transmitters
+
+Link: (Radio Transmitters)[https://www.hackerrank.com/challenges/hackerland-radio-transmitters/problem]
+
+Hackerland is a one-dimensional city with N houses, where each house is located at some point on the axis. The Mayor wants to install radio transmitters on the roofs of the city's houses. Each transmitter has a range D, meaning it can transmit a signal to all houses D units of distance away.
+
+Given a map of Hackerland and the value of D, can you find and print the minimum number of transmitters needed to cover every house in the city? (Every house must be covered by at least one transmitter). Each transmitter must be installed on top of an existing house.
```

Comment:
Below the description (and the Source information) could you write on how to run the tests? The Python template already has some information, you could append that here.